README.md

```diff
@@ -1,30 +1,36 @@
-# @artsy/palette [![CircleCI](https://circleci.com/gh/artsy/palette.svg?style=shield)](https://circleci.com/gh/artsy/palette) [![npm version](https://badge.fury.io/js/%40artsy%2Fpalette.svg)](https://www.npmjs.com/package/@artsy/palette) [![Netlify Status](https://api.netlify.com/api/v1/badges/beb9e8d7-10cc-4a2e-99bb-0d4c6f46db82/deploy-status)](https://app.netlify.com/sites/artsy-palette/deploys)
+# Artsy's Design System 
```

Comment:
Please don't change any of this

Reply:
concur

Reply:
It is similar to how we structure things on force 🤔

<img width="1039" alt="Screenshot 2022-12-08 at 18 21 13" src="https://user-images.githubusercontent.com/36475005/206520556-107cbb15-d8db-4d35-8d04-df740c08f2a5.png">

Is there something you don't like?

Reply:
Ah, i see the meta (and badges) have been moved down, so i'm ok with it now

Reply:
Eh, I find having the badges at the top more useful for a package like this. Also this project is called `@artsy/palette` not `Artsy's Design System`

Reply:
All of our bullet points have been blown out of the meta section into subsections. Feel free to add sub-sections further down the readme, but don't mess with meta (or add to it, concisely, as other meta sections (like force) have done)

Reply:
To me the actual links related to palette feel like more important, but in any case they are both above fold view and the user can directly see them. It also looks better to have some text after h1 instead of having a meta tag directly after which is a h2 as h1 and h2 both have an underline which makes it look funky. 
I did have it like that in the beginning but then i moved it. Didn't think it would be an issue as we also have it in force like that

Reply:
Also, meta has been moved down.  And please add `@artsy/palette` back to the top of the page as that's our NPM lib name, which is important to elevate to the top, as we've done with our other NPM libraries.

Reply:
(Basically, this stuff was fine as it was)

Reply:
It looked like that https://github.com/artsy/palette/blob/27dd48f5177aab6dfd842f2f95ffa41ba31071b9/README.md

Reply:
ah, if we can move that stuff into meta that would be good 👍